setup.py

```diff
@@ -295,7 +295,8 @@ def run_build():
             "Programming Language :: Cython",
             "Topic :: Software Development :: Code Generators",
             "Topic :: Software Development :: Compilers",
-            "Topic :: Software Development :: Libraries :: Python Modules"
+            "Topic :: Software Development :: Libraries :: Python Modules",
+            "Typing :: Typed"
```

Comment:
It seems to me that we can call a package typed only when its internal code uses type annotations / comments.

Reply:
Remember that these are user facing, for the user `Cython` has typings.

Reply:
Well, to be honest, users can still use and some do use parts of the internal code.